be/src/olap/task/index_builder.cpp

```diff
@@ -298,6 +298,33 @@ Status IndexBuilder::_add_nullable(const std::string& column_name,
         }
         return step;
     };
+    if (field->type() == FieldType::OLAP_FIELD_TYPE_ARRAY) {
```

Comment:
there is some old code for field->type() == FieldType::OLAP_FIELD_TYPE_ARRAY bellow.